docs/docusaurus/versioned_docs/version-1.6.X/orc8r/deploy_using_juju.md

```diff
@@ -0,0 +1,110 @@
+---
+id: version-1.6.X-deploy_using_juju
+title: Deploy Orchestrator using Juju
+hide_title: true
+original_id: deploy_using_juju
+---
+
+# Deploy Orchestrator using Juju
```

Comment:
Compared to `docs/readmes/orc8r/deploy_using_juju.md`: Why is this feature for 1.6 not in Beta?

Reply:
@voisey Is it possible to show a docs page only in selected releases?

A simple way of making sure that it is not displayed in 1.7 and 1.8 would be to have only a disclaimer on this page for these versions that it is not (yet) supported.

Reply:
FYI you can preview how it will appear once published https://github.com/magma/magma/wiki/Contributing-Documentation#viewing-changes-locally